lib/src/components/NxThreatIndicatorLegend/NxThreatIndicatorLegend.scss

```diff
@@ -24,6 +24,10 @@
 
   .nx-threat-indicator-legend__threat-container {
     @include nx-container-helpers.container-horizontal;
+
+    .nx-threat-indicator-legend__threat-desc {
+      @include nx-text-helpers.semi-bold;
```

Comment:
Could probably just be added to `.nx-threat-indicator-legel__threat-container`

Reply:
> Could probably just be added to `.nx-threat-indicator-legel__threat-container`

You're right. Saves from having to assign a class to the `span` as well.
Changed here: 27962f7ed6db34d91ac81c933d82b3dadc6ef66f